### PR TITLE
[MilkmApp] Handle cancel transactions

### PR DESCRIPTION
**Pull request by yvesfracari**
https://www.loom.com/share/a39dc3c6b1744cfbabd0a1fa86ea6400

[transaction explorer](https://goerli.etherscan.io/tx/0x8b613e8bd6283d87a9687b90ae9e42f8010bb9593f381845e7369318babac4d9)

### CODE REVIEW

File: apps/cow-tools/src/app/milkman/(components)/ordersTable/CancelButton.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          tokenAddressToSell: swap.tokenIn?.id,
+          tokenAddressToBuy: swap.tokenOut?.id,
+          toAddress: swap.to,
+          amount: BigInt(swap.tokenAmountIn),
+          priceChecker: swap.priceChecker,
+          priceCheckerData: swap.priceCheckerData,
+        }) as MilkmanCancelArgs,
+    );
+
+    await sendTransactions(transactionsData);
```
Comment (luizakp) [inline apps/cow-tools/src/app/milkman/(components)/ordersTable/CancelButton.tsx:33]:
```suggestion
    const cancelTransactionsData = transaction.swaps.map(
      (swap) =>
        ({
          type: TRANSACTION_TYPES.MILKMAN_CANCEL,
          contractAddress: swap.orderContract,
          tokenAddressToSell: swap.tokenIn?.id,
          tokenAddressToBuy: swap.tokenOut?.id,
          toAddress: swap.to,
          amount: BigInt(swap.tokenAmountIn),
          priceChecker: swap.priceChecker,
          priceCheckerData: swap.priceCheckerData,
        }) as MilkmanCancelArgs,
    );

    await sendTransactions(cancelTransactionsData);
```

File: apps/cow-tools/src/app/milkman/(components)/ordersTable/TransactionInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { priceCheckersArgumentsMapping } from "#/lib/priceCheckersMappings";
+import { truncateAddress } from "#/utils/truncate";
+
+export function TransactionInfo({
+  order,
+}: {
+  order: AllTransactionFromUserQuery["users"][0]["transactions"][0]["swaps"][0];
+}) {
+  const priceChecker = getPriceCheckerFromAddressAndChain(
+    order.chainId as 5,
```
Comment (luizakp) [inline apps/cow-tools/src/app/milkman/(components)/ordersTable/TransactionInfo.tsx:20]: is this right?

File: apps/cow-tools/src/app/milkman/[network]/HomePageWrapper.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import { Spinner } from "#/components/Spinner";
 import WalletNotConnected from "#/components/WalletNotConnected";
 import { getNetwork } from "#/contexts/networks";
 
-import { OrderTable } from "../(components)/OrdersTable";
+import { OrderTable } from "../(components)/ordersTable";
```
Comment (luizakp) [inline apps/cow-tools/src/app/milkman/[network]/HomePageWrapper.tsx:13]: would rename the folder as `OrdersTable` instead of `ordersTable`

**Comment by luizakp** [comment]
notice two things:
- that I have to reload the app to sync the new status 
- the transaction url is till the create transaction

we can solve this on this PR or create new tasks for it

File: apps/cow-tools/src/app/milkman/(components)/ordersTable/TransactionInfo.tsx (hunk 2 above)
Comment (yvesfracari) [reply apps/cow-tools/src/app/milkman/(components)/ordersTable/TransactionInfo.tsx:20]: I have a task to change it all to multiple chains. At the end of this week, I will have to solve this for one issue.

**Pull request merged**